<?xml version="1.0" ?>
<launch>

    <node name="realsense_node" pkg="realsense_camera" type="realsense_camera_node" output="screen">
    </node>
    

</launch>
